src/mocks/handlers.ts

```diff
@@ -18,6 +22,8 @@ setupWorker(
   rest.get("/mesos/master/state-summary", fx("marathon-1-task/summary")),
   rest.get("/metadata", fx("dcos/metadata")),
   rest.get("/navstar/lashup/key", (_, res, ctx) => res(ctx.json({}))),
+
+  rest.get("/service/marathon/v2/groups", fx("marathon-1-task/groups")),
```

Comment:
these changes enable local development on the `serviceCreateForm` while also fixing a memory leak (we were requiring the same fixtures again and again. `require`d stuff stays in memory 😊)